install/helm/open-match/subcharts/open-match-scale/templates/scale-backend.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ .Values.scaleBackend.hostName }}
+  namespace: {{ .Release.Namespace }}
+  annotations: {{- include "openmatch.chartmeta" . | nindent 2 }}
+  labels:
+    app: {{ template "openmatch.name" . }}
+    component: scaleBackend
+    release: {{ .Release.Name }}
+spec:
+  selector:
+    app: {{ template "openmatch.name" . }}
+    component: scaleBackend
+  ports:
+  - name: http
+    protocol: TCP
+    port: {{ .Values.scaleBackend.httpPort }}
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.scaleBackend.hostName }}
+  namespace: {{ .Release.Namespace }}
+  annotations: {{- include "openmatch.chartmeta" . | nindent 2 }}
+  labels:
+    app: {{ template "openmatch.name" . }}
+    component: scaleBackend
+    release: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.scaleBackend.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "openmatch.name" . }}
+      component: scaleBackend
+  template:
+    metadata:
+      namespace: {{ .Release.Namespace }}
+      annotations:
+        {{- include "openmatch.chartmeta" . | nindent 8 }}
+      labels:
+        app: {{ template "openmatch.name" . }}
+        component: scaleBackend
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
+        {{- include "openmatch.volumes.configs" . | nindent 8}}
```

Comment:
It seems like this chart is not in sync with the latest helm changes, do you mind updating this file with the redis [volume](https://github.com/googleforgames/open-match/blob/master/install/helm/open-match/templates/backend.yaml#L82) & [volumemount](https://github.com/googleforgames/open-match/blob/master/install/helm/open-match/templates/backend.yaml#L89)?

Reply:
None of these components use redis - do I still need the changes?

Reply:
Oh it should be good then